src/main/java/core/basesyntax/service/impl/CreateReportServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.CreateReportService;
+import java.util.stream.Collectors;
+
+public class CreateReportServiceImpl implements CreateReportService {
+    private static final String DESCRIPTION = "fruit,quantity";
+
+    @Override
+    public String createReport() {
+        String report = Storage.warehouse.stream()
```

Comment:
let's create a method that will get all fruits from Storage instead of directly calling it